pinot-common/pom.xml

```diff
@@ -182,6 +182,10 @@
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-compress</artifactId>
     </dependency>
+    <dependency>
```

Comment:
This is already included in the `pinot-spi`, no need to include again (you might need to rebase to the current master)